mistletoe/html_renderer.py

```diff
@@ -102,13 +137,13 @@ def render_raw_text(self, token: span_token.RawText) -> str:
     def render_html_span(token: span_token.HTMLSpan) -> str:
         return token.content
 
-    def render_heading(self, token: block_token.Heading) -> str:
-        template = '<h{level}>{inner}</h{level}>'
+    def render_heading(self, token: block_token.Heading, attr: Attributes = None) -> str:
+        template = '<h{level}{attr}>{inner}</h{level}>'
         inner = self.render_inner(token)
-        return template.format(level=token.level, inner=inner)
+        return template.format(level=token.level, inner=inner, attr=self._render_attributes(attr))
 
-    def render_quote(self, token: block_token.Quote) -> str:
-        elements = ['<blockquote>']
+    def render_quote(self, token: block_token.Quote, attr: Attributes = None) -> str:
```

Comment:
I didn't add it for `render_paragraph` because depending on `self._suppress_ptag_stack[-1]` it does not set set any tag, in which case it would just ignore `attr`, which I'd find confusing.